src/crimson/osd/pg.h

```diff
@@ -427,6 +427,9 @@ class PG : public boost::intrusive_ref_counter<
   bool is_primary() const final {
     return peering_state.is_primary();
   }
+  bool is_non_primary() const {
+    return peering_state.is_nonprimary();
```

Comment:
I wonder whether we should fix or preserve the naming as is. How about wrapping literally?

Reply:
okay. i found it took me more time to parse "nonprimary" than "non_primary".

changed as suggested.